src/commands/notify.yml

```diff
@@ -67,11 +67,12 @@ parameters:
     type: boolean
     default: true
 
-  dashboard_link:
-    description: The URL to the dashboard
+  test_results_button_links:
```

Comment:
Can we just call it button links??? They may not be test results

Reply:
done